### PR TITLE
Clean common

### PR DESCRIPTION
**Pull request by BakkerJesse**
PR for cleaned Common section,

This tackles the following items identified last week (see slack)

1. Connect ro:Plan and ro:Planobject to nen3610 ontology
1b. Add requirement that instances of ro:Plan als be of type void:Dataset in shapes.
2. Create ro:Planobject and add subClass links

5. Convert all ro:Instruments to skos:Concepts, and create skos:Collections for the enumerations
6. Remove uses of ro:RuimtelijkInstrument
7. Add subclasses of ro:Ondergrond as per the Ondergronden enumerations

### CODE REVIEW

**Comment by eknutov** [comment]
it concerns duplicate and redundant definitions as discussed earlier, as well some parts have been reviewed on Wed. some have not been but we'll review those anyway the next week, so no need to leave it hanging as a PR, just push it on the master so that we have a clean baseline and less conflicts as of Mon-Tue changes if any

File: Models/common/verwijzing/model.ttl [hunk 1]
```diff
@@ -20,256 +20,261 @@
...
+ro:vervangt a owl:ObjectProperty ;
   rdfs:subPropertyOf ro:externPlan ;
   rdfs:isDefinedBy :ro ;
-  dcterms:subject ro_beg:vervangt ;
+  dcterms:subject ro_beg:vervangt
 .
 
-ro:uitTeWerkenIn
-  rdf:type owl:ObjectProperty ;
+ro:uitTeWerkenIn a owl:ObjectProperty ;
```
Comment (pmaria) [inline Models/common/verwijzing/model.ttl]: weird casing here

File: Models/common/verwijzing/model.ttl [hunk 2]
```diff
@@ -20,256 +20,261 @@
...
-  rdf:type owl:Class ;
+ro:Bijlage a owl:Class ;
   rdfs:label "Bijlage"@nl ;
   rdfs:subClassOf ro:Tekst ;
+  rdfs:isDefinedBy :ro
 .
 
-ro:Bijlage_Besluitdocument
-  rdf:type owl:Class ;
+ro:Bijlage_Besluitdocument a owl:Class ;
```
Comment (pmaria) [inline Models/common/verwijzing/model.ttl]: Why are we using underscores here instead of camel casing?

File: Models/common/verwijzing/model.ttl [hunk 3]
```diff
@@ -20,256 +20,261 @@
...
+ro:uitgewerktIn a owl:ObjectProperty ;
   rdfs:subPropertyOf ro:externPlan ;
   rdfs:isDefinedBy :ro ;
-#  dcterms:subject ro_beg: ;
+  dcterms:subject ro_beg:uitgewerktIn
 .
 
-ro:gebruiktInformatieUit
-  rdf:type owl:ObjectProperty ; #does this IRI deviate too much from 'informatie in extern plan/besluit'?
+ro:gebruiktInformatieUit a owl:ObjectProperty ; #does this IRI deviate too much from 'informatie in extern plan/besluit'?
```
Comment (pmaria) [inline Models/common/verwijzing/model.ttl]: property name is fine, as long as we can match the original enum value via the skos:notation of the corresponding skos:Concept.

File: Models/common/verwijzing/shapes.ttl [hunk 4]
```diff
@@ -68,27 +71,29 @@ ro_str:Teksttype_BP a sh:NodeShape ;
...
 .
 
-ro_str:Teksttype_PSV rdf:type sh:NodeShape ;
+ro_str:Teksttype_PSV a sh:NodeShape ;
   sh:property [
     sh:path rdf:type ;
     sh:in (
       ro:Beleid
-      ro:Beleid_Gemandateerd_aan_GS #TODO: adjusted to Naming Conventions
+      ro:BeleidGemandateerdAanGS
```
Comment (pmaria) [inline Models/common/verwijzing/shapes.ttl]: Now this doesn't match with the weird underscored classes.

File: Models/common/verwijzing/model.ttl (hunk 1 above)
Comment (BakkerJesse) [reply Models/common/verwijzing/model.ttl]: Why, it should map to 'uit te werken in' ? I agree it is not aesthetically pleasing, though.

File: Models/common/verwijzing/model.ttl (hunk 2 above)
Comment (BakkerJesse) [reply Models/common/verwijzing/model.ttl]: I revised this part in a new commit

**Pull request merged**